README.md

```diff
@@ -1,30 +1,27 @@
 # SIG-Release
 
 Welcome to SIG-Release, home of release and feature management.
-
-## Onboarding
-
-Please check out our general [GitHub Onboarding Guide](docs/Eclipse-Tractus-X-GitHub-getting-started.md) and 
-further information can be found here in the [getting started page](https://eclipse-tractusx.github.io/docs/oss/getting-started) on our web-page. For PI11 Planning please have a look into [PI11-Consortia-Planning](docs/PI11-Planning.md).
+If you are new to this repository, you might find the [getting started guide](docs/getting-started.md) useful.
+If you are interested in the current state of planning, have a look at our [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
 
 ## The planning and roadmap process
 
-We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/Open-Refinement-and-Planning-in-Open-Source-Communities.md).
+We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
 
 While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
-has its own issue management, the [release planning board](https://github.com/orgs/eclipse-tractusx/projects/26)
+has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
 is used to align the overarching Tractus-X releases.
 
 ### How can I get involved
 
 In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
-feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-project-management/issues/new/choose) and describe your request.
+feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
 Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
 
 Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
```

Comment:
Maybe we can also mention our new open meeting page?

Reply:
Done! What do you think?  [a54dcb9](https://github.com/eclipse-tractusx/sig-release/pull/478/commits/a54dcb97e2dfce3e05bca8f1c8e6bb3cb5fd8e83)

Reply:
love it thx